src/hooks/postMutations.js

```diff
@@ -0,0 +1,17 @@
+import { gql } from "@apollo/client";
+
+export const ADD_CART_MUTATION = gql`
+  mutation AddCartMutation(
+    $userId: ID!
+    $storeItemId: ID!
+    $quantity: Int!
+  ) {
+    createUserStoreItem(userId: $userId, storeItemId: $storeItemId, quantity: $quantity) {
+      id
+      userId
+      storeItemId
+      quantity
+    }
+    errors
```

Comment:
Nice work adding errors. Make sure to provide feedback to users if error accrues.